src/Refact.hs

```diff
@@ -53,9 +54,10 @@ refactorPath rpath = do
                        , "<https://github.com/mpickering/apply-refact>"
                        ]
 
-runRefactoring :: FilePath -> FilePath -> FilePath -> String -> IO ExitCode
-runRefactoring rpath fin hints opts =  do
+runRefactoring :: FilePath -> FilePath -> FilePath -> [Extension] -> [Extension] -> String -> IO ExitCode
+runRefactoring rpath fin hints ys ns opts =  do
```

Comment:
Perhaps `enabled disabled` as the names? I appreciate that `ys`/`ns` was meant to signify that, but it wasn't immediately obvious to me.